lib/sycamore/sycamore/data/element.py

```diff
@@ -69,14 +76,17 @@ def __init__(
         title: Optional[str] = None,
         columns: Optional[list[str]] = None,
         rows: Optional[list[Any]] = None,
+        table: Optional[Table] = None,
+        tokens: Optional[list[dict[str, Any]]] = None,
         **kwargs,
     ):
         super().__init__(element, **kwargs)
         self.data["type"] = "table"
-        self.data["properties"] = {}
         self.data["properties"]["title"] = title
         self.data["properties"]["columns"] = columns
         self.data["properties"]["rows"] = rows
+        self.data["table"] = table
```

Comment:
It is maybe a little bit awkward to have both the table property and the existing columns/rows properties. Eventually we should rewrite everything else in terms of table, but I don't think we want to modify existing code quite yet. Open to suggestions.

Reply:
What do you mean by this?

Reply:
My next step is to support converting HTML tables to the `Table` class. I started on that, but it's a bit tricky, so I deferred it to a later PR b/c it's not necessary for the blog post.

Reply:
sorry, a bit confused myself also. I thought you try to get rid of columns/rows, and use table instead? For html tables, do we have such use case? I thought it was more on into html, but not from html.

Reply:
One of my goals is to have a common representation of a table across different formats. That way, for example, when we come up with a good way to represent a table for embedding, we can use it independently of the source format. I think that ideally we would have people access rows and columns via the Table interface, rather than storing them separately for some formats in the Element.